script.js

```diff
@@ -0,0 +1,432 @@
+$(document).ready(function () {
+
+    var checkIfSpell = true;
+    var classCheck = "";
+    //get all the inputs from the localstorage
+    if (localStorage.length != 0) {
+        //hit point local storage
+        if (localStorage.getItem("hitPoint") != "undefined") {
```

Comment:
Try not to retrieve from local storage twice since there is a performance penalty.  You could do something like

```
var hitPoint = localStorage.getItem('hitPoint');
if (hitPoint !== 'undefined')
```

Also, `getItem('name')` takes 1 argument, not 2, so the second argument is not used
```